File: src/Client/StorageArea.ts
/**
 * Identifies the web storage area.
 */
export const StorageArea = Object.freeze({

	/**
	 * Indicates the local storage.
	 */
	Local: "Local",

	/**
	 * Indicates the session storage.
	 */
	Session: "Session"
});

/**
 * Identifies the web storage area.
 */
export type StorageArea = typeof StorageArea[keyof typeof StorageArea];

File: src/Server/StorageArea.cs
namespace Belin.UI.UI;

using System.Text.Json.Serialization;

/// <summary>
/// Identifies the web storage area.
/// </summary>
[JsonConverter(typeof(JsonStringEnumConverter))]
public enum StorageArea {

	/// <summary>
	/// Indicates the local storage.
	/// </summary>
	Local,

	/// <summary>
	/// Indicates the session storage.
	/// </summary>
	Session
}
